extension/src/background/messageListener/freighterApiMessageListener.ts

```diff
@@ -85,6 +85,7 @@ export const freighterApiMessageListener = (
     browser.windows.create({
       url: chrome.runtime.getURL(`/index.html#/grant-access?${encodeOrigin}`),
       ...WINDOW_SETTINGS,
+      width: 400,
```

Comment:
needed to make the window a little wider to accomodate design changes to the buttons.